src/PairEvaluatorHertz.h

```diff
@@ -2,27 +2,49 @@
 // Copyright (c) 2021-2024, Auburn University
 // Part of azplugins, released under the BSD 3-Clause License.
 
-/*!
- * \file PairEvaluatorHertz.h
- * \brief Defines the pair force evaluator class for Hertz potential
- */
-
 #ifndef AZPLUGINS_PAIR_EVALUATOR_HERTZ_H_
 #define AZPLUGINS_PAIR_EVALUATOR_HERTZ_H_
 
 #include "PairEvaluator.h"
 
-#ifdef NVCC
+#ifdef __HIPCC__
 #define DEVICE __device__
 #else
 #define DEVICE
 #endif
 
+namespace hoomd
+    {
 namespace azplugins
     {
-
 namespace detail
     {
+
+struct PairParametersHertz : public PairParameters
+    {
+#ifndef __HIPCC__
+    PairParametersHertz() : epsilon(0) { }
+
+    PairParametersHertz(pybind11::dict v, bool managed = false)
```

Comment:
Calling this `v` is very non-descriptive, but since it is so localized, I don't mind/care. Are there other pair potentials where we could look to make the name of the dictionary consistent? Or is `v` the consistent name we are using throughout?- if so, fine

Reply:
`v` is used by HOOMD and by @mphoward in the base class

Reply:
Ok, great, then we will use `v` consistently everywhere.